WordPress/Makefile

```diff
@@ -0,0 +1,10 @@
+
+dependencies: xcframeworks
+
+
+xcframeworks:
+	curl https://cdn.a8c-ci.services/gutenberg-mobile/Gutenberg-v1.121.0.tar.gz --output Frameworks/Gutenberg.tar.gz -C -
+	tar -xvf Frameworks/Gutenberg.tar.gz -C Frameworks/ -k
+	mv -n Frameworks/Frameworks/*.xcframework Frameworks/
+	rm -rf Frameworks/Frameworks Frameworks/dummy.txt
+	mkdir -p Frameworks/hermes.xcframework/ios-arm64/dSYMs Frameworks/hermes.xcframework/ios-arm64_x86_64-simulator/dSYMs
```

Comment:
Given `gutenberg-mobile` should have an hopefully short time left in the project, I think this rudimental way to fetch it is acceptable.

Also, the version is not likely to change at all, which again makes me comfortable with simply hardcoding it there and duplicating it it in Fastlane.

Reply:
Is it possible to use `binaryTarget` to import gutenberg-mobile via SPM?

Reply:
If not, can we use the existing Rakefile, instead of adding a new Makefile?

Reply:
> Is it possible to use `binaryTarget` to import gutenberg-mobile via SPM?

I haven't tried using SwiftPM directly yet. I inherited this setup from #23750 and it worked 😄 How about trying it as a followup?

> If not, can we use the existing Rakefile, instead of adding a new Makefile?

Yeah, that's a good point. I'll take care of it.